tests/functional/pv/pv_services/test_pvc_creation_in_megabytes.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import logging
+from ocs_ci.framework.testlib import bugzilla, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames="interface",
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHFILESYSTEM], marks=pytest.mark.polarion_id("OCS-323")
```

Comment:
please add the correct polarion id

Reply:
done. -->@pytest.mark.polarion_id("OCS-5476")